packages/insured-bridge-relayer/src/ProfitabilityCalculator.ts

```diff
@@ -121,7 +124,7 @@ export class ProfitabilityCalculator {
     slowRevenue: BN,
     speedUpRevenue: BN,
     instantRevenue: BN
-  ): RelaySubmitType {
+  ): { relaySubmitType: RelaySubmitType; profitabilityInformation: string } {
```

Comment:
this main function now returns a string of `profitabilityInformation` which is appended to both profitable and unprofitable relays